fingertip/plugins/script.py

```diff
@@ -31,7 +35,36 @@ def run(m, scriptpath, cache=0, no_unseal=False):
         m.ssh.upload(scriptpath, dst=scriptname)
         m.log.debug(f'uploaded {scriptpath} into {scriptname}')
         m(f'chmod +x {scriptname}')
+        if mode == 'test':
```

Comment:
may be easier to `m = m.apply('.hooks.disable_proxy')` outside, in test(). maybe not, I haven't tried.